clusterctl/cmd/root.go

```diff
@@ -37,12 +37,13 @@ var RootCmd = &cobra.Command{
 
 func Execute() {
 	if err := RootCmd.Execute(); err != nil {
-		glog.Exit(err)
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
 	}
 }
 
 func exitWithHelp(cmd *cobra.Command, err string) {
-	glog.Error(err)
+	fmt.Fprintln(os.Stderr, err)
```

Comment:
Why fmt instead of glog?

Reply:
glog inserts dynamic time based, output that we can't compare and is generally a bad experience for CLI users (imagine if kubectl was outputting help / error instructions for basic commands using glog. For example, you will see output like this (look at the last line):

```
    		Global Flags:
    		      --alsologtostderr                  log to standard error as well as files
    		      --log-flush-frequency duration     Maximum number of seconds between log flushes (default 5s)
    		      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
    		      --log_dir string                   If non-empty, write log files in this directory
    		      --logtostderr                      log to standard error instead of files (default true)
    		      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
    		  -v, --v Level                          log level for V logs
    		      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging
    		
    		Use "clusterctl delete [command] --help" for more information about a command.
    		
    		F0530 15:41:04.806689   20340 root.go:41] unknown flag: --invalid-flag
```

Which we cannot string match against. While we could go around it, I started thinking about the user experience of seeing such odd output with the CLI, it doesn't match the other output, and I think it is better to not have it.